ui/shared/utils/constants.js

```diff
@@ -291,8 +291,8 @@ export const INDIVIDUAL_FIELD_NOTES = 'notes'
 export const INDIVIDUAL_FIELD_PROBAND_RELATIONSHIP = 'probandRelationship'
 
 export const INDIVIDUAL_FIELD_CONFIGS = {
-  [FAMILY_FIELD_ID]: { label: 'Family ID' },
-  [INDIVIDUAL_FIELD_ID]: { label: 'Individual ID' },
+  [FAMILY_FIELD_ID]: { label: 'Family ID', description: 'Family ID of the individual' },
+  [INDIVIDUAL_FIELD_ID]: { label: 'Individual ID', description: 'ID of the Individual' },
```

Comment:
I do not want this changing on the other pages. You need to create custom constants to pass to the form, not change the existing constants. So for example in the `LoadWorkspaceDataForm` you might define something like

```
const FIELD_DESCRIPTIONS = {
  [FAMILY_FIELD_ID]:  'Family ID of the individual',
  [INDIVIDUAL_FIELD_ID]: 'ID of the Individual', //change to include that should match the VCF ID
}
const REQUIRED_FIELDS = INDIVIDUAL_ID_EXPORT_DATA.map(config => (
  {  ...config, description: FIELD_DESCRIPTIONS[config.field] })
```